code/src/reducers/game.js

```diff
@@ -0,0 +1,78 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const game = createSlice({
+	name: 'game',
+	initialState: {
+		username: null,
+		started: false,
```

Comment:
I'm intrigued about this started state, I don't see what it's being used for, but I feel inspired to maybe use it in my own project and double down on the loading component because ours never showed up!